app/livechat/lib/LivechatRoomType.js

```diff
@@ -85,6 +85,10 @@ export default class LivechatRoomType extends RoomTypeConfig {
 		}
 	}
 
+	allowMemberAction(room, action) {
+		return action === RoomMemberActions.INVITE || RoomMemberActions.JOIN;
```

Comment:
I guess you want to test if `action` is either `INVITE` OR `JOIN` right? in this case, this code doesn't do that.. what you could do is something like:
```suggestion
		return [RoomMemberActions.INVITE, RoomMemberActions.JOIN].includes(action);
```